go.mod

```diff
@@ -127,7 +126,6 @@ require (
 	github.com/eapache/go-resiliency v1.2.0 // indirect
 	github.com/elastic/elastic-agent-client/v7 v7.0.0-20210727140539-f0905d9377f6 // indirect
 	github.com/elastic/go-concert v0.2.0 // indirect
-	github.com/elastic/go-licenser v0.4.0 // indirect
```

Comment:
I am curious about this one, probably they change the way they check the license

Reply:
nice catch, it was removed by `go mod tidy` since we use it in the mage file and it is ignored by the build